src/app/api/user/[id]/route.js

```diff
@@ -0,0 +1,68 @@
+import {NextResponse} from 'next/server'
```

Comment:
```suggestion
import { NextResponse } from 'next/server'
```
Checa los errores de eslint